src/OrchardCore.Modules/OrchardCore.OpenApi/OrchardCore.OpenApi.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+  <PropertyGroup>
+    <TargetFramework>$(AspNetCoreTargetFramework)</TargetFramework>
+    <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
```

Comment:
do we need?

Reply:
Will need later for implementing authentication.